contents/docs/feature-flags/common-questions.mdx

```diff
@@ -25,6 +25,27 @@ Local evaluation payloads are typically used for much, _much_ more than 10 users
 
 ---
 
+## My feature flags are too expensive, what can I do about this?
+
+Feature flags can get expensive if your code ends up sending multiple requests to get feature flags for each person.
+
+If this is happening on the client side, i.e. while using the posthog-js SDK, you have two options:
+
+1. As per the [advanced configuration](/docs/libraries/js#advanced-configuration), set the `advanced_disable_feature_flags_on_first_load` setting. This stops automatically requesting feature flags whenever a page loads. This is useful when you're immediately calling `posthog.identify()` anyway, which requests flags again.
```

Comment:
```suggestion
1. As per the [advanced configuration](/docs/libraries/js#advanced-configuration), set `advanced_disable_feature_flags_on_first_load` setting to `true`. This stops automatically requesting feature flags whenever a page loads. This is useful when you're immediately calling `posthog.identify()` anyway, which requests flags again.
```

Reply:
```suggestion
1. Using the the [advanced configuration](/docs/libraries/js#advanced-configuration), set the `advanced_disable_feature_flags_on_first_load` setting to `true`. This stops automatically requesting feature flags whenever a page loads. This is useful when you're immediately calling `posthog.identify()` anyway, which requests flags again.
```

`As per` sounds a bit like I'm being talked down to.